composer/trainer/trainer.py

```diff
@@ -2977,6 +2977,23 @@ def save_checkpoint(
             weights_only=weights_only,
         )
 
+    def save_checkpoint_to_save_folder(self):
+        """Checkpoints the training :class:`~.State` using a CheckpointSaver if it exists.
+
+        Args:
+            None
+
+        Raises:
+            ValueError: If ``_checkpoint_saver`` does not exist.
+
+        Returns:
+            None
+        """
+        if self._checkpoint_saver == None:
```

Comment:
```suggestion
        if self._checkpoint_saver is None:
```
both are correct, but `is None` is the generally recommended style